src/styles/App.css

```diff
@@ -17,140 +22,188 @@
 }
 
 body {
-  font-family: 'Poppins', sans-serif;
-  line-height: 1.4;
-  color: #333;
-  background: var(--softwhite);
-  font-weight: 400;
+  background: var(--pale-grey);
 }
 
 /* -------------------------------------------NAVBAR */
 
 nav {
-  height: 58px;
-  background: var(--mainWhite);
-  top: 0;
-  left: 0;
-  right: 0;
   display: flex;
-  gap: 2rem;
-  padding-left: 50px;
-  padding-right: 50px;
   align-items: center;
-  width: 100%;
-  box-shadow: 0 1px 3px 0 rgba(0, 0, 0, 0.07), 0 1px 2px 0 rgba(0, 0, 0, 0.05);
+  justify-content: space-between;
+  margin-left: 6.25rem;
+  margin-right: 6.25rem;
+  height: 5.938rem;
+  padding: 1.438rem 6.188rem 1.688rem 6.25rem;
+  background-color: var(--white-Two);
 }
 
-.logo {
-  text-decoration: none;
+.nav-center {
+  display: flex;
+  align-items: center;
 }
 
-.logo h2 {
-  color: var(--blueColor);
+.logo {
+  height: 2.313rem;
+  margin-right: 2.938rem;
+  font-family: var(--montserrat);
+  font-size: 1.875rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: var(--azure);
+  text-decoration: none;
+  width: auto;
 }
 
-nav > ul {
+.menu {
   display: flex;
-  gap: 2rem;
+  gap: 2.563rem;
   align-items: center;
   list-style: none;
-  height: 100%;
-  color: var(--linkColor);
 }
 
-.active {
-  color: #8a4baf;
+.nav-link {
+  color: var(--linkColor);
+  text-decoration: none;
+  font-size: 0.813rem;
+  letter-spacing: 1.9px;
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  cursor: pointer;
+  height: 1rem;
+  font-family: var(--montserrat);
+  opacity: 0.5;
 }
 
-nav > ul > li {
-  cursor: pointer;
+.active {
+  opacity: 1;
 }
 
-.nav-link {
-  color: var(--clearBlack);
-  text-decoration: none;
-  font-size: inherit;
+.oval {
+  width: 2.813rem;
+  height: 2.813rem;
+  padding: 0.875rem;
+  border: solid 1px #e8e8e8;
 }
 
 main {
-  padding-left: 50px;
-  padding-right: 50px;
-  padding-top: 50px;
-  width: 100%;
+  margin-left: 6.25rem;
+  margin-right: 6.25rem;
+  padding-top: 2.375rem;
+  padding-left: 6.25rem;
+  padding-right: 6.25rem;
```

Comment:
- [ ] Good job so far 👏 👏 In addition to what you have done, I would suggest that you use the Circular progress bar in order to match the design given. Here is the comparison 👇 
Given design
![image](https://user-images.githubusercontent.com/90222110/230646700-8883e703-d922-42b3-a2da-dfe804b18c31.png)

Your design
![image](https://user-images.githubusercontent.com/90222110/230646791-f7a3fae7-a732-4db4-ad40-ce2da3eb1d26.png)

If you don't want to build it yourself, you can use a package and there are a lot of such packages out there.
One of them I would suggest is [this one here](https://www.npmjs.com/package/react-circular-progressbar)

Reply:
> * [ ]  Good job so far 👏 👏 In addition to what you have done, I would suggest that you use the Circular progress bar in order to match the design given. Here is the comparison 👇
>   Given design
>   ![image](https://user-images.githubusercontent.com/90222110/230646700-8883e703-d922-42b3-a2da-dfe804b18c31.png)
> 
> Your design ![image](https://user-images.githubusercontent.com/90222110/230646791-f7a3fae7-a732-4db4-ad40-ce2da3eb1d26.png)
> 
> If you don't want to build it yourself, you can use a package and there are a lot of such packages out there. One of them I would suggest is [this one here](https://www.npmjs.com/package/react-circular-progressbar)

Thanks @BertrandConxy for your providing me link to the npm circular svg package. I have no added the circular progress bar.